### PR TITLE
Completed view-products-for-sales-front-end | Peter Han

### PR DESCRIPTION
**Pull request by peterjhan88**
Initial app.jsx changed to render Header and ProductList
custom styles.css added for fonts and sizes of div
unused class deleted
image

### CODE REVIEW

File: client/components/header.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default class Header extends React.Component {
+  render() {
+    return (
+      <header className='bg-dark d-flex header-height flex-justify-start align-items-center'>
+        <div className='ml-5 mr-2 d-flex align-items-center text-weight-bold text-white header-logo'>$</div>
+        <div className='d-flex align-items-center text-white header-name'>Wicked Sales</div>
+      </header>
```
Comment (thebearingedge) [inline client/components/header.jsx:9]: 🤔 Can you simplify this? There are _a lot_ of CSS classes here.

File: client/components/product-list-item.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export default class ProductListItem extends React.Component {
+  render() {
+    return (
+      <div className='col-3 mx-4 my-3 d-flex align-items-center card'>
+        <img src={this.props.imgUrl} alt="product Image" className='card-img-top image-contain' />
+        <div className='card-body'>
+          <h5 className='card-title'>{this.props.productName || 'Product Name'}</h5>
+          <p className='card-text text-price text-weight-bold'>{`$ ${(this.props.price / 100).toFixed(2)}` || '0.00'}</p>
+          <p className='card-text'>{this.props.shortDescription || 'Short description sit amet, consectetur adipsicing elit...'}</p>
```
Comment (thebearingedge) [inline client/components/product-list-item.jsx]: Are these `||` guards necessary?

File: client/components/product-list.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          productName={product.name}
+          price={product.price}
+          shortDescription={product.shortDescription}
+        />
+      );
+    });
+    return (
+      <div className='col-12 d-flex flex-wrap justify-content-center bg-light'>
+        {allProducts}
+      </div>
```
Comment (thebearingedge) [inline client/components/product-list.jsx:41]: Try using a `.row` here and then `.col-md-4` on each list item.

File: server/public/styles.css [hunk 4]
```diff
@@ -0,0 +1,20 @@
+* {
+  box-sizing: border-box;
+}
```
Comment (thebearingedge) [inline server/public/styles.css]: This `box-sizing` reset is already included with Bootstrap.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: server/public/styles.css (hunk 4 above)
Comment (peterjhan88) [reply server/public/styles.css]: deleted

File: client/components/product-list-item.jsx (hunk 2 above)
Comment (peterjhan88) [reply client/components/product-list-item.jsx]: deleted

File: client/components/product-list.jsx (hunk 3 above)
Comment (peterjhan88) [reply client/components/product-list.jsx:41]: changed as requested

File: client/components/header.jsx (hunk 1 above)
Comment (peterjhan88) [reply client/components/header.jsx:9]: simplified by using custom css

**Pull request merged**